webapp/src/api-interface.d.ts

```diff
@@ -1,3 +1,8 @@
+interface PaymentForm {
+  email: string
```

Comment:
Wspominałem ostatnio że sam request do płatności już nie będzie miał emaila, bo wezmę go sobie z osobnego requesta. Obstawiam że nie jest to scope tego taska ale tak tylko wspominam.